x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/text_fields_body.tsx

```diff
@@ -63,6 +68,10 @@ export const TextFieldsBody: React.FC = () => {
           </EuiTableRowCell>
           <EuiTableRowCellCheckbox>
             <EuiCheckbox
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.snippetArialLabel',
```

Comment:
Oh wait no sorry, I lied, there's a few more.
```suggestion
                'xpack.enterpriseSearch.appSearch.engine.resultSettings.table.snippetAriaLabel',
```